playbooks/openshift/delete-cluster.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- hosts: seed-hosts
+  roles:
+  - roles/openshift-ansible-contrib/openshift-pv-cleanup
```

Comment:
Actually, no `roles` - see this one for an example:
https://github.com/redhat-cop/casl-ansible/blob/master/playbooks/openshift/delete-openstack.yml#L7